animal/models/animal.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class Animal(models.Model):
+    _name = "animal"
+    _description = "Animal"
+    _inherit = ["mail.thread", "mail.activity.mixin"]
+    _order = "name"
+
+    name = fields.Char()
+    ref = fields.Char(string="Reference")
+    species_id = fields.Many2one("animal.species", string="Species", required=True)
+    breed_id = fields.Many2one("animal.breed", string="Breed", required=True)
+    color_id = fields.Many2one("animal.color", string="Color")
+    size = fields.Char()
+    weight = fields.Float(string="Weight (in kg)")
```

Comment:
Hi @Deriman-Alonso , I think we should take advantage of the opportunity to remove the specification regarding the pet's weight in kilograms. I believe the best approach would be to do something similar to how products/pickings handle weight based on the overall adjustment. 
![image](https://github.com/OCA/partner-contact/assets/69461150/e0b42e4a-d62c-4178-9a14-ce66954e9940)
![image](https://github.com/OCA/partner-contact/assets/69461150/e510723c-d238-4c90-ac2f-e51d8b9b4236)

Reply:
Hi @Christian-RB , as far as my understanding goes, those UoM settings comes from the "product" module. Would it be properly to make the "animals" module depend on  "product", or rather create its own menu in the settings, even tho it means there could be different types of UoM for the weight?

Reply:
Oh You are right @Deriman-Alonso, in that case I assume is better to keep it the same way. Thanks!